docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,100 @@ sidebar_position: 2
 
 # Quick Start
 
-In this section we are providing a sample application that one can deploy and just use to play around.
+In this quick start, you will see a sample application that you can deploy and
+use to play around.  As part of this exercise, a backing service and an
```

Comment:
I think in the intro, we should highlight the problem we are trying to solve.
Maybe, we can build a small diagram, showing the two different components of the application.
We can at that time explain, what usually a developer would do and the potential issues with that approach.

Reply:
Done. 22659064cbf6055a24cbb747dd9e4f5e978037f8